frontend/routes/auth/settings/lock-section.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, ExternalLink } from '@vegaprotocol/ui-toolkit'
+import { useForm } from 'react-hook-form'
+import { useCallback } from 'react'
+import { useJsonRpcClient } from '../../../contexts/json-rpc/json-rpc-context'
+import { useNavigate } from 'react-router-dom'
+import { FULL_ROUTES } from '../../../routes/route-names'
+import config from '../../../lib/config'
+import { SettingsSection } from './settings-section'
+import { RpcMethods } from '../../../lib/client-rpc-methods'
+
+export const locators = {
+  settingsLockButton: 'settings-lock-button',
+  settingsFeedbackDescription: 'settings-feedback-description',
+  settingsFeedbackLink: 'settings-feedback-link'
+}
+
+export const LockSection = () => {
+  const { client } = useJsonRpcClient()
+  const navigate = useNavigate()
+  const { handleSubmit: handleLock } = useForm()
+  const lock = useCallback(async () => {
```

Comment:
Might be missing full context here, but I feel like some of the useCallback's in this PR might not be needed. https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse